Makefile

```diff
@@ -274,6 +274,15 @@ build/$(TEST_JAR): $(TEST_SOURCES) build/$(CONVERTER_JAR)
 	$(JAVAC) -source $(JAVA_TARGET) -target $(JAVA_TARGET) -Xlint:-options -cp "build/jar/*:build/converter/*" -d build/test $(TEST_SOURCES)
 	$(JAR) cf $@ -C build/test .
 
+.ONESHELL:
```

Comment:
Make this explicit by combining below commands into one (`&&`)

Reply:
b4fa8c55677bec5245a312cfe917d28d481d3fd7